test/cases/compile_errors/access_non-existent_member_of_error_set.zig

```diff
@@ -0,0 +1,12 @@
+const Foo = error{A};
+comptime {
+    const z = Foo.Bar;
+    _ = z;
+}
+
+// error
+// backend=stage2
```

Comment:
Just to confirm you are targeting the right backend: `stage2` implies incremental, while `llvm` implies stage2 LLVM. I now since how that can get confusing. I will redo the `Backend` to include `stage2_llvm` and `stage2_incremental` instead, and once both get close to finish, we can consolidate the two into one `stage2` or something.

Reply:
I don't think it'll matter too much for compile error tests but I did test them without the llvm backend just to be sure. If you think it's more trouble than it's worth I can change them to use `backend=llvm` instead.

Reply:
Sadly, it will be a problem on non-x86_64 hosts such as aarch64 where some lowerings such as `airRetLoad` are not yet implemented. Otherwise, I agree it should be no difference. Anyhow, leave it as-is for now. I just wanted to double check. I will investigate some more next why the tests are failing for you. In the meantime, here's first improvement to the harness: https://github.com/ziglang/zig/pull/11952